### PR TITLE
Putting JES' package object on a diet

### PR DESCRIPTION
**Pull request by geoffjentry**

### CODE REVIEW

**Comment by geoffjentry** [comment]
@gauravs90 @cjllanwarne

**Comment by geoffjentry** [comment]
Ugh, I've got something weird going on between my git, intellij and/or github. not everything is agreeing what's in what file.

**Comment by geoffjentry** [comment]
Ahh, intellij decided to lose my VCS root in supportedBackends for the 213626th time today

**Comment by cjllanwarne** [comment]
Sure, why not! 👍

File: supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesCallPaths.scala [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    JesCallPaths(rootPath, workflowRootPath, callRootPath, stdoutFilename, stderrFilename, jesLogFilename, returnCodeFilename)
+  }
+}
+
+case class JesCallPaths private(rootPath: Path, workflowRootPath: Path, callRootPath: Path, stdoutFilename: String, stderrFilename: String, jesLogFilename: String, returnCodeFilename: String) {
+  lazy val returnCodePath: Path = callRootPath.resolve(returnCodeFilename)
+  lazy val stdoutPath: Path = callRootPath.resolve(stdoutFilename)
+  lazy val stderrPath: Path = callRootPath.resolve(stderrFilename)
+  lazy val callContext = new CallContext(callRootPath, stdoutFilename, stderrFilename)
+}
```
Comment (gauravs90) [inline supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesCallPaths.scala]: New line 😝

**Comment by gauravs90** [comment]
👍

**Pull request merged**